05_syntax_lab/04.py

```diff
@@ -1,8 +0,0 @@
-"""
-Write a program that reads lines from the user
-until an empty line is inserted.
-After the user typed in an empty line,
-print all previously inserted lines in reverse
-order (from last to first)
-"""
-
```

Comment:
Tthat's an empty file.
In any case you might want to move on to the next set of exercises of the first week.